src/content/blog/2023-01-10-eslint-2022-year-review.md

```diff
@@ -0,0 +1,112 @@
+
+1. [Part 1](https://eslint.org/blog/2022/08/new-config-system-part-1/) explains why we needed a new configuration system and the problems with the current system.
+2. [Part 2](https://eslint.org/blog/2022/08/new-config-system-part-2/) explains the new configuration system and the thinking that went into its design.
+3. [Part 3](https://eslint.org/blog/2022/08/new-config-system-part-2/) explains how to use the developer preview of the configuration system.
+
+Since we published that third post, the new configuration system is almost complete and is available for use by end users ([see the docs](https://eslint.org/docs/latest/user-guide/configuring/configuration-files-new) for more).
+
+We anticipate the final release of the new configuration system in 2023 along with a transition period where both the new and old configuration systems can be used before the old is eventually removed.
+
+## Updates to support ES2022
+
+As we do each year, the ESLint team stays on top of the latest ECMAScript versions to make sure ESLint continues to support the more recent JavaScript syntax and features. This year, we added support for:
+
+1. [Top-level await](https://github.com/tc39/proposal-top-level-await)
+1. [Class fields](https://github.com/tc39/proposal-class-fields)
+1. [Ergonomic brand checks for private fields](https://github.com/tc39/proposal-private-fields-in-in)
+
+You can enable these in your configuration by setting `ecmaVersion: 2022` or `ecmaVersion: "latest"`.
+
+## Project kickoffs
+
+In addition to our normal maintenance and development, we also kicked off a couple of additional projects:
+
+1. **Documentation Refresh** - we hired a technical writer to start reviewing, organizing, and rewriting our documentation. We have never had a documentation-focused project and we felt like 2022 was the right time to start this project given the funds we had available. We anticipate this project to continue until mid-2023.
+2. **ESLint Community** - the strength of ESLint is in its ecosystem and we had been searching for a way to better organize that ecosystem. As a result, we created the [`eslint-community`](https://github.com/eslint-community) GitHub organization as a home for high-value projects in the ESLint ecosystem. We intend this to be a place for projects related to ESLint to live and thrive so that they never fall out of maintenance. We are still working through the details on this and you can expect a followup blog post soon.
+
+## Income review
+
+During 2022, ESLint received $197,267.73 USD from all of our income sources. You can see the breakdown of our income sources in the following table.
+
+| **Source** | **Amount** |
+|-----------|-------------|
+| Open Collective | $127,741.96 |
+| GitHub Sponsors | $26,606.61 |
+| Tidelift | $29,088.00 |
+| Carbon Ads | $13,831.16 |
+| Stackaid.us | $77.54 |
```

Comment:
This $77.54 isn't included in the total. Also, `Stackaid.us` appears as a link, not sure if that's intentional.

Reply:
Wow, nice catch! My Google Sheet didn't catch the extra row after I broke it out.
Stackaid.us is being auto-linked by Markdown. Not sure how to avoid that.